src/main/kotlin/reactor/kotlin/extra/math/MathFluxExtensions.kt

```diff
@@ -57,8 +58,11 @@ inline fun <reified T:Number> Flux<T>.sumAll(): Mono<T> =
  *
  * @author Simon Baslé
  * @since 1.0.0
- * @deprecated Please use sumLong() as a direct replacement of consider more general purpose sumAll()
+ * @deprecated Please use sumLong() as a direct replacement, or consider more general purpose sumAll().
```

Comment:
dang. it's now `sumAsLong` isn't it? deprecated messages need to be updated (doc, annotation message and annotation replaceWith) here and in the other similar deprecations

Reply:
resolved